src/test/kotlin/domain/MineBoardTest.kt

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+
+class MineBoardTest : FunSpec({
+    test("지뢰찾기 보드를 만든다.") {
+        val minePositions: List<Position> = listOf(
+            Position(1, 1),
+            Position(2, 2),
+            Position(3, 3),
+        )
+
+        val mineBoard = MineBoard.init(
+            height = 3,
+            width = 3,
+            mineCount = 3,
```

Comment:
```suggestion
            mineCount = minePositions.size,
```
마인 갯수를 얻어서 넣는 것이 좋을 것 같아요!